storage/aptosdb/src/lib.rs

```diff
@@ -1988,11 +2014,7 @@ impl DbWriter for AptosDB {
             )?;
 
             // Delete the genesis transaction
-            LedgerPruner::prune_genesis(
-                self.ledger_db.clone(),
-                self.state_store.clone(),
-                &mut batch,
-            )?;
+            LedgerPruner::prune_genesis(self.ledger_db.clone(), &mut batch)?;
```

Comment:
curious why we need to prune ledger, statekv while not seeing state_merkel's genesis being pruned?